### DISCUSSION TITLE
Can't load plugin: sqlalchemy.dialects:cassandra.sqlalchemy_cassandra

### DISCUSSION THREAD

**Discussion by qianze10078** [sqlalchemy/sqlalchemy#9500] [Usage Questions]
engine_url = '{name}+{module}://{username}:{password}@{host}:{port}/{db}'
name = 'cassandra'
module = 'sqlalchemy_cassandra'

when i use sqlalchemy1.3.5,everything is ok,but when i use sqlalchemy1.4.16,It reported the following error:【Can't load plugin: sqlalchemy.dialects:cassandra.sqlalchemy_cassandra】。

please tell me what exactly is cassandra dialect in version 1.4.16 of sqlalchemy?

**Comment by qianze10078** [comment]
help me!

**Comment by CaselIT** [comment]
Hi,

You should ask to the Cassandra dialect developers. It not maintained by sqlalchemy

**Reply by qianze10078** [reply]
ok,thank you!
